server/src/test/java/io/crate/integrationtests/DDLIntegrationTest.java

```diff
@@ -95,6 +97,23 @@ public void testCreateTable() throws Exception {
         assertEquals(1L, response.rowCount());
     }
 
+    @Test
+    @UseNewCluster
+    public void oid_value_is_restored_after_cluster_restart() throws Exception {
```

Comment:
I think the new `PersistedClusterStateServiceTests` test case is already enough and tests all we need. Such, I'd remove this test.